data/json/monsters/reptile_amphibian.json

```
@@ -1,4 +1,52 @@
 [
+  {
+    "id": "mon_frog",
+    "type": "MONSTER",
+    "name": { "str": "bullfrog" },
+    "description": "The American bullfrog, in its natural habitat.  It feeds on insects, mice, lizards and any other living thing it can stuff down its gullet.",
+    "default_faction": "frog",
+    "bodytype": "frog",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "AMPHIBIAN" ],
+    "volume": "750 ml",
+    "weight": "1 kg",
+    "hp": 3,
+    "speed": 80,
+    "material": [ "flesh" ],
+    "symbol": "b",
+    "color": "green",
+    "melee_cut": 0,
+    "dodge": 6,
+    "reproduction": { "baby_egg": "egg_frog", "baby_count": 8, "baby_timer": 26 },
+    "harvest": "mammal_tiny",
+    "special_attacks": [ { "type": "leap", "cooldown": 10, "max_range": 5 } ],
```

```suggestion
    "special_attacks": [ { "type": "leap", "cooldown": 4, "max_range": 5, "allow_no_target": true } ],
```
Basing cooldown on other critters with ~the same range and "allow_no_target": true.